app/src/main/java/com/example/oddhours/data/repository/JobRepository.kt

```diff
@@ -91,7 +102,8 @@ class JobRepository() {
     }
 
     fun getShiftsForUIList(): MutableList<ShiftsListModel> {
-        val jobModelList = buildJobList() // TODO: maybe at this point we don't need to call buildJobList()? just use the jobModelList?
+        val jobModelList =
+            buildJobList() // TODO: maybe at this point we don't need to call buildJobList()? just use the jobModelList?
```

Comment:
i dont mind moving code to new line to improve readability like we're doing for long function parameters, but i'm not sure if there's a need here?

Reply:
sorry i am not following what you mean here?

Reply:
it looks like the only change here is moving the rest of the code after the '=' sign to a new line?